keyboards/splitkb/aurora/corne/keymaps/x123/keymap.c

```diff
@@ -0,0 +1,444 @@
+    */
+    const char *text = PSTR("abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ123456789!@#$%^&*()[]{}-=_+?");
+
+    if (is_keyboard_master()) {
+        oled_write_P(text, false);
+    } else {
+        oled_write_P(text, false);
+    }
+    return false;
+}
+
+bool process_record_user(uint16_t keycode, keyrecord_t *record) {
+  switch (keycode) {
+    case QWERTY:
+      if (record->event.pressed) {
+        set_single_persistent_default_layer(_QWERTY);
+        layer_move(_QWERTY);
+      }
+      return false;
+      break;
+    case QWERTY_NOHOMEROW:
+      if (record->event.pressed) {
+        set_single_persistent_default_layer(_QWERTY_NOHOMEROW);
+        layer_move(_QWERTY_NOHOMEROW);
+      }
+      return false;
+      break;
+    case COLEMAK:
+      if (record->event.pressed) {
+        set_single_persistent_default_layer(_COLEMAK);
+        layer_move(_COLEMAK);
+      }
+      return false;
+      break;
+    case COLEMAK_NOHOMEROW:
+      if (record->event.pressed) {
+        set_single_persistent_default_layer(_COLEMAK_NOHOMEROW);
+        layer_move(_COLEMAK_NOHOMEROW);
+      }
+      return false;
+      break;
+    case LOWER:
+      if (record->event.pressed) {
+        layer_on(_LOWER);
+        update_tri_layer(_LOWER, _RAISE, _ADJUST);
+      } else {
+        layer_off(_LOWER);
+        update_tri_layer(_LOWER, _RAISE, _ADJUST);
+      }
+      return false;
+      break;
+    case RAISE:
+      if (record->event.pressed) {
+        layer_on(_RAISE);
+        update_tri_layer(_LOWER, _RAISE, _ADJUST);
+      } else {
+        layer_off(_RAISE);
+        update_tri_layer(_LOWER, _RAISE, _ADJUST);
+      }
+      return false;
+      break;
+    case ONESHOT:
+      if (record->event.pressed) {
+        set_oneshot_layer(_ONESHOT, ONESHOT_START);
+      } else {
+        clear_oneshot_layer_state(ONESHOT_PRESSED);
+      }
+      return false;
+      break;
+    case DT_UP_50:
+      if (record->event.pressed) {
+        g_tapping_term += 50;
+      }
+      return false;
+      break;
+    case DT_DOWN_50:
+      if (record->event.pressed) {
+        g_tapping_term -= 50;
+      }
+      return false;
+      break;
+    case DT_UP_X2:
+      if (record->event.pressed) {
+        g_tapping_term *= 2;
+      }
+      return false;
+      break;
+    case DT_DOWN_X2:
+      if (record->event.pressed) {
+        g_tapping_term /= 2;
+      }
+      return false;
+      break;
+    case DT_200:
+      if (record->event.pressed) {
+        g_tapping_term = 200;
+      }
+      return false;
+      break;
+    case DT_MAX:
+      if (record->event.pressed) {
+        g_tapping_term = 34464;
+      }
+      return false;
+      break;
+  }
+  return true;
+}
+#else
+bool process_record_user(uint16_t keycode, keyrecord_t *record) {
+  switch (keycode) {
+    case QWERTY:
+      if (record->event.pressed) {
+        set_single_persistent_default_layer(_QWERTY);
+        layer_move(_QWERTY);
```

Comment:
This isn't needed. 

The `set_single_persistent_default_layer` function sets the default layer state, making this redundant.

```suggestion
```

Reply:
Thanks for the heads up, removed.